src/components/Button.js

```diff
@@ -2,16 +2,14 @@ import React from 'react';
 import PropsTypes from 'prop-types';
 
 const Button = ({ name, value, onClick }) => (
-  <div>
-    <button
-      type="button"
-      name={name}
-      value={value}
-      onClick={onClick}
-    >
-      {value}
-    </button>
-  </div>
+  <button
+    type="button"
+    name={name}
+    value={value}
+    onClick={onClick}
+  >
+    {value}
+  </button>
```

Comment:
-you're expected to have an incoming `wide` prop [see [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)]
- this `wide` prop should be boolean, validate this with `propTypes` [see [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)]

the aim of this was to make students work on conditional rendering